index.html

```diff
@@ -80,16 +173,172 @@ <h4>Monday February 27th - Friday March 3rd, 2023</h4>
+      <div id="in_person_info">
+
+        <div class="row">
+          <div class="col-sm">
+            <div class="form-group">
+              <label for="lname">Large-font name</label>
+              <input type="text" class="form-control" name="lname" id="lname"
+                placeholder="optional, defaults to first name" maxlength="100">
+            </div>
+            <div class="form-group">
+              <label for="sname">Small-font name</label>
+              <input type="text" class="form-control" name="sname" id="sname"
+                placeholder="optional, defaults to last name" maxlength="100">
+            </div>
+            <div class="form-group">
+              <label for="pronoun">Third-person pronoun</label>
+              <input type="text" class="form-control" name="pronoun" id="pronoun"
+                placeholder="e.g. she/her, he/him, they/them" maxlength="100">
+              <small id="inputPronounHelp" class="form-text text-muted">Third-person pronoun you want people to use to
+                refer to you, such as: they/she/he/ze, or simply just your names.
+                You can also say "no preference" or just enter a space. <a
+                  href="https://confluence.slac.stanford.edu/display/LSSTDESC/Personal+Pronouns" target="_blank">Why ask
+                  for pronouns ?</a></small>
+              <div class="invalid-feedback">Please provide your choice of pronoun.</div>
+            </div>
+          </div>
+          <div class="col-sm">
+            <div id="badge">
+              <div id="lname-d">
+                <span id="lname-s"></span>
+              </div>
+              <div id="sname-d">
+                <span id="sname-s"></span>
+              </div>
+              <div id="affili-d">
+                <span id="affili-s"></span>
+              </div>
+              <div id="pronoun-d">
+                <span id="pronoun-s"></span>
+              </div>
+              <div id="event-d">
+                <span id="event-s">Collaboration Meeting, July 24-28 2023</span>
+              </div>
+              <div id="location-d">
+                <span id="location-s">SLAC National Accelerator Laboratory</span>
+              </div>
+              <div id="logo1" class="logo"></div>
+              <div id="logo2" class="logo"></div>
+            </div>
+          </div>
+        </div>
+
+
+
+        <div class="container">
+
+
+          <h5>COVID Safety</h5>
+          <p>Please perform COVID testing prior to the start of the event. SLAC will
+           not ask for proof of vaccination nor negative COVID test from participants.
+	   <b>Note:</b>  For all attendees who arrive by air travel, indoor masking is
+	   required within three days of arrival.  Attendees who arrive via international
+	   air travel will be required to wear a disposable mask (surgical, KN95, KF94, i
+	   or N95) in the indoor sessions at SLAC  within five days post-travel.</p>
```

Comment:
same comment here. Use of "within" seems strange and possibly confusing (could mean wear a mask once in those 5 days).